spec/intl.html

```diff
@@ -1338,6 +1338,101 @@ <h1>CalendarEraYear ( _calendar_, _dateLike_ )</h1>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardatedayofweek">
+          <h1>CalendarDateDayOfWeek ( _calendar_, _date_ )</h1>
+          <p>The abstract operation CalendarDateDayOfWeek takes a date-like object _date_ and returns the day of the week in the calendar represented by _calendar_.</p>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardatedayofyear">
+          <h1>CalendarDateDayOfYear ( _calendar_, _date_ )</h1>
+          <p>The abstract operation CalendarDateDayOfYear takes a date-like object _date_ and returns the day of the year in the calendar represented by _calendar_.</p>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardateweekofyear">
+          <h1>CalendarDateWeekOfYear ( _calendar_, _date_ )</h1>
+          <p>The abstract operation CalendarDateWeekOfYear takes a date-like object _date_ and returns the week of the year in the calendar represented by _calendar_.</p>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardatedaysinweek">
+          <h1>CalendarDateDaysInWeek ( _calendar_, _date_ )</h1>
+          <p>The abstract operation CalendarDateDaysInWeek takes a date-like object _date_ and returns the number of days in the given week in the calendar represented by _calendar_.</p>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardatedaysinmonth">
+          <h1>CalendarDateDaysInMonth ( _calendar_, _date_ )</h1>
+          <p>The abstract operation CalendarDateDaysInMonth takes a date-like object _date_ and returns the number of days in the given month in the calendar represented by _calendar_.</p>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardatedaysinyear">
+          <h1>CalendarDateDaysInYear ( _calendar_, _date_ )</h1>
+          <p>The abstract operation CalendarDateDaysInYear takes a date-like object _date_ and returns the number of months in the given year in the calendar represented by _calendar_.</p>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardatemonthsinyear">
+          <h1>CalendarDateMonthsInYear ( _calendar_, _date_ )</h1>
+          <p>The abstract operation CalendarDateMonthsInYear takes a date-like object _date_ and returns the number of months in the given year in the calendar represented by _calendar_.</p>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardateinleapyear">
+          <h1>CalendarDateInLeapYear ( _calendar_, _date_ )</h1>
+          <p>The abstract operation CalendarDateInLeapYear takes a date-like object _date_ and returns *true* if it falls within a leap year for the calendar represented by _calendar_ and *false* otherwise.</p>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardatefields">
+          <h1>CalendarDateFields ( _calendar_, _fields_ )</h1>
```

Comment:
There should probably be more commentary about what this operation does and what implementations are supposed to do with it. See https://github.com/tc39/proposal-temporal/issues/1899#issuecomment-962859406 for more info.

Reply:
I'll read through and add more commentary, thanks!